torpaste.py

```diff
@@ -43,6 +47,7 @@ def new_paste():
         if (request.form['content']):
             status, message = logic.create_new_paste(
                 request.form['content'],
+                request.form['visibility'],
```

Comment:
See my comment above. Handling paste metadata more generically will probably prevent most issues of that kind.